.github/workflows/nightly_run.yml

```diff
@@ -644,14 +644,11 @@ jobs:
         with:
           os: ${{ matrix.os }}
 
-      # It is a temporary solution until new wasi-sdk that includes bug fixes is released
-      - name: build wasi-libc from source
+      - name: download and install wasi-sdk
         if: matrix.test_option == '$WASI_TEST_OPTIONS'
-        run: |
-          git clone https://github.com/WebAssembly/wasi-libc
-          cd wasi-libc
-          make -j AR=/opt/wasi-sdk/bin/llvm-ar NM=/opt/wasi-sdk/bin/llvm-nm CC=/opt/wasi-sdk/bin/clang THREAD_MODEL=posix
-          echo "SYSROOT_PATH=$PWD/sysroot" >> $GITHUB_ENV
+        uses: ./.github/actions/install-wasi-sdk-wabt
```

Comment:
why do you run install-wasi-sdk-wabt twice?

Reply:
oops. fixed